jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/Pool.java

```diff
@@ -13,58 +13,25 @@
 
 package org.eclipse.jetty.util;
 
-import java.io.Closeable;
-import java.io.IOException;
-import java.util.ArrayList;
 import java.util.Collection;
-import java.util.Collections;
-import java.util.List;
-import java.util.Objects;
-import java.util.concurrent.CopyOnWriteArrayList;
-import java.util.concurrent.ThreadLocalRandom;
-import java.util.concurrent.atomic.AtomicInteger;
 import java.util.function.Function;
-import java.util.function.ToIntFunction;
+import java.util.stream.Stream;
 
 import org.eclipse.jetty.util.annotation.ManagedAttribute;
 import org.eclipse.jetty.util.annotation.ManagedObject;
-import org.eclipse.jetty.util.component.Dumpable;
-import org.eclipse.jetty.util.component.DumpableCollection;
-import org.eclipse.jetty.util.thread.AutoLock;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 /**
- * <p>A pool of objects, with optional support for multiplexing,
- * max usage count and several optimized strategies plus
- * an optional {@link ThreadLocal} cache of the last release entry.</p>
- * <p>When the method {@link #close()} is called, all {@link Closeable}s
- * object pooled by the pool are also closed.</p>
+ * <p>A pool of objects, with support for multiplexing and several
+ * optimized strategies plus an optional {@link ThreadLocal} cache
```

Comment:
The strategies are no longer relevant to the `Pool` interface. They should be moved to `ConcurrentPool`